android/java/res/layout/approve_tx_bottom_sheet.xml

```diff
@@ -141,15 +141,53 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/amount_fiat" />
 
+    <Space
+        android:id="@+id/tab_top_space"
+        android:layout_width="0dp"
+        android:layout_height="8dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/tv_approve_dialog_additional_details"
+        app:layout_constraintBottom_toTopOf="@+id/warning_container" />
+
+    <LinearLayout
+        android:id="@+id/warning_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="12dp"
+        android:background="#32E32444"
```

Comment:
@deeppandya, this color code is part of the legacy code and I was not able to find it in the exported tokens. I've reused the same implementation adopted for the warning message in the "sign message" ([Here](https://github.com/brave/brave-core/blob/05f3e182766fe9bb801182c647f35b69574b0e32/android/java/res/layout/fragment_sign_tx_message.xml#L154-L190)). I doubt we will use it again in the future. Let me know if you prefer to handle it differently.

Reply:
let's keep it as is. i am going to push a new change in leo soon.